### ISSUE TITLE
Duplicated warning message for not enough cells

### ISSUE THREAD

**Issue by Marius1311** [theislab/cellrank#174]
I call `g_fwd.compute_metastable_states(cluster_key='clusters', n_states=11)`, which prints out:

```
Computing metastable states
INFO: Using pre-computed schur decomposition
WARNING: The following groups have less than requested number of cells (30): `Ngn3 high EP_1`
Adding `.schur_vectors`
       `.metastable_states`
       `.coarse_T`
       `.coarse_stationary_distribution`
    Finish (0:00:03)
WARNING: The following groups have less than requested number of cells (30): `Ngn3 high EP_1
```

Can we get rid of the duplication?

**Comment by michalk8** [comment]
@Marius1311 it should be already fixed and merged on master? Do you have the current version?

**Comment by michalk8** [comment]
I get only following:
```python
Computing metastable states
WARNING: The following groups could not be mapped uniquely: `Alpha_1, Alpha_2`
WARNING: The following groups have less than requested number of cells (400): `Alpha_2, Ngn3 low EP`
Adding `.schur_vectors`
       `.metastable_states`
       `.coarse_T`
       `.coarse_stationary_distribution`
    Finish (0:00:00)
```
Different warnings, so they should be fine.

**Comment by Marius1311** [comment]
Okay, never mind then, will try this again!

**Comment by Marius1311** [comment]
If not fixed already, will be fixed by #184.

**Comment by michalk8** [comment]
Should be fixed. Closing this.
